xml/System.Text.Json/JsonElement.xml

```diff
@@ -615,11 +675,14 @@
         <Parameter Name="value" Type="System.Text.Json.JsonElement" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="propertyName">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="propertyName">Name of the property to find.</param>
+        <param name="value">Receives the value of the located property.</param>
+        <summary>Looks for a property named <paramref name="propertyName" /> in the current object, returning whether or not such a property existed. When the property exists <paramref name="value" /> is assigned to the value of that property.</summary>
+        <returns><see langword="true" /> if the property was found, <see langword="false" /> otherwise.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.InvalidOperationException"> This value's <see cref="P:System.Text.Json.JsonElement.Type" /> is not <see cref="F:System.Text.Json.JsonValueType.Object" />.</exception>
```

Comment:
nit: Is it fine to have a space here? If so, then please ignore. Otherwise, there are a few (~11 places) where we should remove the space between the exception tag and the start of the message.

```suggestion
        <exception cref="T:System.InvalidOperationException">This value's <see cref="P:System.Text.Json.JsonElement.Type" /> is not <see cref="F:System.Text.Json.JsonValueType.Object" />.</exception>
```

Reply:
```suggestion
        <exception cref="T:System.InvalidOperationException">This value's <see cref="P:System.Text.Json.JsonElement.Type" /> is not <see cref="F:System.Text.Json.JsonValueType.Object" />.</exception>
```

Reply:
I don't think it's a problem, but we can make the quick change via suggestions.